src/ingestion/views.py

```
@@ -0,0 +1,15 @@
+from rest_framework import generics
+from .models import TicketEvent
+from .serializers import TicketEventSerializer
+
+class TicketEventIngestView(generics.CreateAPIView):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)